public/global.css

```diff
@@ -16,9 +16,10 @@
 }
 
 body {
+	display: flex;
+  flex-direction: column;
```

Comment:
indention looks messed up here: some are tabs, some spaces. Please unify.

Reply:
Yeah, wanted to convert all files to spaces and update the vscode rules. But will do it once you accept, just to avoid formatting changes covering the real changes.

Reply:
reasonable